package.json

```diff
@@ -62,8 +61,9 @@
   "dependencies": {
     "@azure/cosmos": "^3.17.1",
     "@pagopa/express-azure-functions": "^2.0.0",
-    "@pagopa/io-functions-commons": "^26.3.0",
+    "@pagopa/io-backend-notifications-sdk": "x",
     "@pagopa/io-functions-admin-sdk": "^0.32.1",
```

Comment:
@StefanoDoc not your fault (it's actually on me 😅), but we can improve the repo if we use `x` as version for `@pagopa/io-functions-admin-sdk` as well